models/schemas/reviewSchema.ts

```diff
@@ -34,3 +36,17 @@ const ReviewSchema = new mongoose.Schema<Review>({
 });
 
 export default ReviewSchema;
```

Comment:
Need to also handle editing and deleting for reviews (decreasing/incresaing a score, or removing the score entirely)

Reply:
I do have a ReviewSchema updateOne middleware hook